_src/_posts/2017-04-24-learning-rtb.markdown

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
###Reference must be replaced with ##Reference to be aligned with the formatting of old blog posts